domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/ResponseUtils.java

```diff
@@ -59,4 +60,15 @@ public static boolean isAdmissionResponse(Response response) {
         ResponseType responseType = response.getResponseType();
         return responseType == ResponseType.FULL_ADMISSION || responseType == ResponseType.PART_ADMISSION;
     }
+
+    public static boolean isFullDefenceDisputeAndNoMediation(Response response) {
```

Comment:
Missing unit test for this method

Reply:
Okay so these methods require individual unit tests, even though they are already being called in the other unit tests in `DefendantResponseNotificationServiceTest.java`? @sabahirfan

Reply:
Yes please, as the other unit test which are calling the same methods will not cover the negative test scenarios.